cli/src/generate/render.rs

```diff
@@ -1346,7 +1346,7 @@ impl Generator {
         add_line!(self, "#endif");
         add_line!(self, "");
         add_line!(self, "#ifdef _WIN32");
-        add_line!(self, "#define TS_PUBLIC __declspec(dllexport)");
+        add_line!(self, "#define TS_PUBLIC");
```

Comment:
Do we need this whole `TS_PUBLIC` thing at all? I don't remember what issue it solved.

Reply:
it's so if anyone decides to compile with `-fvisibility=hidden` the symbols are *still* exported anyways. sometimes it's nice to reduce the noise/visible symbols from the lib or parser, which might stem from the enums/symbol names in parser.c.